extensions/smallrye-opentracing/runtime/pom.xml

```diff
@@ -46,7 +46,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-undertow</artifactId>
+            <artifactId>quarkus-resteasy</artifactId>
```

Comment:
I think at some point we should really avoid a strong dependency on RESTEasy. Using OpenTracing should be doable even if you don't have REST services.

But I suppose it can wait.

Reply:
That makes sense, but is probably an edge case that will need more thought and

Reply:
@gsmet, indeed, tracing non-HTTP requests would be very useful too